backend-tests/tests/test_create_org.py

```diff
@@ -80,7 +80,8 @@ def test_success(self, clean_migrated_mongo):
             )
 
         # get the tenant id (and verify that only one tenant exists)
-        r = tenantadmi.call("GET", tenantadm.URL_INTERNAL_TENANTS)
+        qs = parse.urlencode({"q": tenant})
+        r = tenantadmi.call("GET", tenantadm.URL_INTERNAL_TENANTS + "?" + qs)
```

Comment:
Because the staging environment is full of tenants, and this query will return all of them :)
The q parameter is the one used by the admin panel, and it is a filter on thee name of the tenant.